app/app.go

```diff
@@ -341,7 +341,6 @@ func New(
 
 	app.PaymentKeeper = *paymentmodulekeeper.NewKeeper(
 		appCodec,
-		app.BankKeeper,
```

Comment:
This was prematurely added over a year ago to do fee burning. Since we won't be doing fee burning until we have some form of immediate execution, this is being removed